adoc/admin-updates.adoc

```diff
@@ -17,6 +17,23 @@ Run `sudo zypper update` on the management workstation before any attempt to upd
 
 Updating of {kube} components is handled via `skuba`.
 
+The general procedure should look like this:
+
+. Update the packages and reboot your management workstation to get all the latest changes to skuba and it's dependencies
+. Run `skuba addon upgrade apply` on the management workstation
+.
+. Check if there are addon upgrades available for the current cluster
```

Comment:
Check if all deployed applications in the cluster are compatible with the next kubernetes version we are upgrading to.
In case a deployment is not compatible update it before you continue, here.